src/Components/UI/videoformmodal/VideoFormModal.jsx

```diff
@@ -1,117 +1,205 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import {
+  Formik,
+  Form,
+  Field,
+  ErrorMessage,
+} from 'formik';
+import { useDispatch } from 'react-redux';
 import MyButton from '../button/MyButton';
-import Input from '../input/Input';
+import { createMovie } from '../../../Services/Handlers/AsyncActionsHendlers';
+import { useFormReuquestSelector } from '../../../Services/Selectors/MoviesSelectors';
+import { setFormRequest } from '../../../Services/Actions/MoviesActions';
 
 const emptyVideo = {
   title: '',
+  voteAverage: undefined,
+  releaseDate: '',
   posterPath: '',
-  year: '',
-  genre: ['horro', 'drama'],
-  rating: '',
+  overview: '',
+  genres: ['horro', 'drama'],
   runtime: '',
-  description: '',
 };
 
-function VideoForm({ editVideo, onSubmit, hideModal }) {
+const isValidUrl = (url) => {
+  let correctUrl;
+  try {
+    correctUrl = new URL(url);
+  } catch (e) {
+    return false;
+  }
+  return correctUrl;
+};
+
+function isIsoDate(str) {
+  if (!/\d{4}-\d{2}-\d{2}/.test(str)) return false;
+  return true;
```

Comment:
NIT: no need in conditions and return, because the extpression returns boolean itself, so you can just
```
return /\d{4}-\d{2}-\d{2}/.test(str);
```